src/priorities/_screen.scss

```diff
@@ -0,0 +1,29 @@
+/*
+ * Priorities (Government of Canada activities and initiatives) (all screen views)
+ */
+
+.gc-nttvs {
+	a {
+		&:active,
+		&:focus,  {
```

Comment:
I am not sure there should be a comma `,` here, it might be complaining about this